staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -946,7 +946,7 @@ const (
 	VirtualMachinePaused VirtualMachineConditionType = "Paused"
 
 	// This condition indicates that the VM was renamed
-	RenameConditionType VirtualMachineConditionType = "Rename Operation"
```

Comment:
ouch, I missed this one.  We're not technically supposed to change things like this because of compatibility, but maybe if there hasn't already been a kubevirt release since this was introduced we'll be okay.

Reply:
The PR that introduced this was merged after the v0.27.0 tag. So it hasn't been released yet.

Reply:
yay!